api.py

```diff
@@ -113,7 +89,7 @@ def reload_data(self):
         '''
         # data
         try:
-            with open(self.users_file, encoding='utf-8') as f:
+            with open(self.data_file, encoding='utf-8') as f:
```

Comment:
mb this whole "reload_data" function actually has no use at all

mind removing it?

Reply:
removed